uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1015.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1015 {
+  
+  /**
+   * Update is_locked value for phone type option group
+   *
+   * @return bool
+   */
+  public function upgrade_1012() {
```

Comment:
oops, missed this the first time. You updated the trait class but didn't change the function name.